canopeum_frontend/src/pages/Register.tsx

```diff
@@ -169,116 +60,150 @@ const Register = () => {
     }
   }
 
+  const invalidFieldClass = 'is-invalid'
```

Comment:
Can this be moved outside the function so it's defined once at top-level ?

You'll likely re-use the same class in other forms, so I'd say you may as well move it to constants now. (src/constants/style.ts is a suggestion). You don't have to do that now in this PR though. Just a suggestion. (do please move it out of the function though)